user-docs/latest/content-manager/managing-relational-fields.md

```diff
@@ -0,0 +1,56 @@
+# Managing relational fields
```

Comment:
I didn't know where to put this comment.
I think it's worth it to mention the blue and green circle's color you can have for relation selection.
When the D&P feature is activated for the content-type of the relation.
Green: Status published
Blue: Status Draft